rest/config.go

```diff
@@ -1023,9 +1023,22 @@ func RunServer(config *ServerConfig) {
 	}
 
 	sc := NewServerContext(config)
-	for _, dbConfig := range config.Databases {
+	bucketUUIDToDbs := make(map[string][]string, len(config.Databases))
+
+	for dbName, dbConfig := range config.Databases {
 		if _, err := sc.AddDatabaseFromConfig(dbConfig); err != nil {
-			base.Fatalf("Error opening database %s: %+v", base.MD(dbConfig.Name), err)
+			base.Fatalf("Error opening database %s: %+v", base.MD(dbName), err)
+		}
+		dbContext, err := sc.GetDatabase(dbName)
```

Comment:
AddDatabaseFromConfig returns the databaseContext as the first parameter (currently ignored above), so you can use that, and remove this call and the subsequent error handling.

Reply:
This change has been committed to the latest revision.